test/integration/typescript-ignore-errors/test/index.test.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-env jest */
+/* global jasmine */
+import { join } from 'path'
+import {
+  renderViaHTTP,
+  nextBuild,
+  findPort,
+  launchApp,
+  killApp,
+  File
+} from 'next-test-utils'
+
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 1000 * 60 * 2
+
+const appDir = join(__dirname, '..')
+const nextConfigFile = new File(join(appDir, 'next.config.js'))
+
+describe('TypeScript with error handling options', () => {
+  for (const ignoreDevErrors of [false, true]) {
+    for (const ignoreBuildErrors of [false, true]) {
```

Comment:
I find it important to test all the 2×2 matrix of options, because otherwise we can miss option spilling to the wrong mode.